CMakeLists.txt

```diff
@@ -441,6 +450,50 @@ endif()
 add_subdirectory(smtk)
 
 ################################################################################
+# Install a package targets file
+#
+# Setup the exports for the library when used from an installed location
+message(STATUS "MOAB_INSTALL_TARGETS: ${MOAB_INSTALL_TARGETS}")
+install(
+  EXPORT SMTK
+  DESTINATION lib/cmake/SMTK
+  FILE SMTKTargets.cmake
+)
+
+export(PACKAGE SMTK)
+
+################################################################################
+# Create a build directory package configuration file
+#
+# Setup the config file for exports that stores what other thirdparty
+# packages we need to search for ( MOAB, Remus, etc ) for when using the
+# build version of smtk
+#
+# Required to happen before we include the docs directory
+set(SMTK_MODULE_DIR "${SMTK_SOURCE_DIR}/CMake")
+set(SMTK_CONFIG_DIR "${PROJECT_BINARY_DIR}")
+configure_file(
+  ${PROJECT_SOURCE_DIR}/CMake/SMTKConfig.cmake.in
+  ${PROJECT_BINARY_DIR}/SMTKConfig.cmake
+  @ONLY)
+
+################################################################################
+# Create an install package configuration file
+#
+# Setup the config file for exports that stores what other thirdparty
+# packages we need to search for ( MOAB, Remus, etc ) for when using the
+# install version of smtk
+set(SMTK_MODULE_DIR "\${CMAKE_CURRENT_LIST_DIR}")
+set(SMTK_CONFIG_DIR "\${CMAKE_CURRENT_LIST_DIR}")
+configure_file(
+  ${PROJECT_SOURCE_DIR}/CMake/SMTKConfig.cmake.in
+  ${PROJECT_BINARY_DIR}/CMakeFiles/SMTKConfig.cmake
+  @ONLY)
+
+install (FILES ${PROJECT_BINARY_DIR}/CMakeFiles/SMTKConfig.cmake
+         DESTINATION lib/cmake/SMTK)
```

Comment:
Should `lib/cmake/SMTK` be `${CMAKE_INSTALL_LIBDIR}/cmake/SMTK`? That way both 32- and 64-bit packages can coexist on Fedora-like distros.

PS. It is OK to say, "No, you are crazy."